clients/reality-data/src/RealityDataClient.ts

```diff
@@ -13,15 +13,15 @@ import { AuthorizedClientRequestContext, getArrayBuffer, getJson, RequestQueryOp
 import { ECJsonTypeMap, WsgInstance } from "./wsg/ECJsonTypeMap";
 import { WsgClient } from "./wsg/WsgClient";
 
-/** Currenlty supported  ProjectWise ContextShare reality data types
+/** Currently supported ProjectWise ContextShare reality data types
  * @internal
  */
-export enum RealityDataType {
-  REALITYMESH3DTILES = "RealityMesh3DTiles", // Web Ready Scalable Mesh
+export enum DefaultSupportedTypes {
```

Comment:
If this is beta we should have the documentation set correctly so it appears in the API. @MarcBedard8 and @EricPaquetB can you provide more explicit description for types. Web Ready Scalable Mesh and Orbit Point Cloud are clear enough but the other would benefit from more explicit descriptions.

Reply:
To add to this, could you guys verify if the type is `"Cesium3DTiles"` or `"Cesium_3DTiles"`? The old code was ambiguous on which one is the expected type name.

Reply:
It is "Cesium3DTiles".  Also, since the reality data type is a string entry in Context Share, we had some entries that was not following the correct case; either camel case or all uppercase. That was the reason why we use a toUppercase in the switch statement  for RealityDataType. If we are going to revert back to string then I think we should also revert back to make a independant type case comparaison.

Reply:
Here are descriptions for various RealityDataType to add for documentation:
RealityMesh3DTiles -> "Web Ready 3D Scalable Mesh",
OPC -> "Web Ready Point Cloud",
TERRAIN3DTILE -> "Web Ready Terrain Scalable Mesh",
OMR -> "Orbit Mapping Resource",
Cesium3DTiles ->  "Cesium 3D Tiles",